apps/clinical/src/components/forms/investigations/__tests__/InvestigationsForm.test.tsx

```
@@ -449,14 +536,94 @@ describe('InvestigationsForm', () => {
+
+      // Simulate an active encounter session with MATCHED reason
+      (useEncounterSessionStore as jest.Mock).mockReturnValue({
+        activeEncounter: { id: 'encounter-uuid-123' },
+        matchReasons: ['MATCHED'],
+        canEditOrCreate: true,
+        isLoading: false,
+      });
+
+      // Simulate backend returning CBC as already ordered in this encounter
+      (getExistingServiceRequestsForAllCategories as jest.Mock).mockResolvedValue(
+        [
```

ESLint

[error] Replace `getExistingServiceRequestsForAllCategories·as·jest.Mock).mockResolvedValue(⏎········` with `⏎········getExistingServiceRequestsForAllCategories·as·jest.Mock⏎······).mockResolvedValue(`

(prettier/prettier)